app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -76,29 +76,36 @@ protected function indexData($request)
         );
     }
 
-    protected function getSubbrowserItems($digitalPublicationId)
+    protected function getBrowserData($prependScope = [])
     {
-        $articles = $this->repository->where('digital_publication_id', $digitalPublicationId)
-                                     ->withDepth()
-                                     ->defaultOrder()
-                                     ->get();
+        $query = $this->repository->withDepth()->defaultOrder();
+
+        $search = $this->request->get('search', $prependScope['search'] ?? null);
+        if ($search) {
+            $query->where('title', 'like', '%' . $search . '%');
```

Comment:
Make this search case-insensetive. When I search for "about" I get no results, but when I search for "About" I get all the about articles.

Reply:
Latest commit looks good!